yapftests/yapf_test.py

```diff
@@ -573,9 +573,9 @@ def g():
 
 
 
-        #comment
```

Comment:
I thought the point of these not changing is that they are outside of the range 3 3.

Presumably this means that your change is not checking for being within the range(s).